Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "tokio-wifiscanner"
-version = "0.2.1"
+version = "0.2.2"
 authors = ["Michał Mania <micmania90@gmail.com>"]
 edition = "2018"
 description = "List WiFi hotspots in your area"
-repository = "https://github.com/mic90/tokio-wifiscanner"
+repository = "https://github.com/DJStompZone/tokio-wifiscanner"
```

Comment:
Hey man, happy to accept any improvements but please leave the crates ownership as it was ;)